coral-CORAL_2_3_21-global-mutex-frontier.patch

```diff
@@ -0,0 +1,47 @@
+--- a/src/FrontierAccess/src/Connection.h	2017-04-28 09:43:29.945972505 -0500
++++ b/src/FrontierAccess/src/Connection.h	2011-03-22 05:36:50.000000000 -0500
+@@ -91,7 +91,7 @@
+       /// The type converter
+       TypeConverter*                                   m_typeConverter;
+       /// The connection lock
+-      static boost::mutex s_lock;
++      mutable boost::mutex m_lock;
```

Comment:
@Dr15Jones,  this is already fixed in coral 2.3.21 https://svnweb.cern.ch/trac/lcgcoral/browser/coral/tags/CORAL_2_3_21/FrontierAccess/src/Connection.h#L94
https://svnweb.cern.ch/trac/lcgcoral/browser/coral/tags/CORAL_2_3_21/FrontierAccess/src/Connection.cpp?rev=21698#L31
In fact all changes in this patch are already part of coral 2.3.21.